src/sdl2/surface.rs

```diff
@@ -414,7 +416,8 @@ impl SurfaceRef {
         }
     }
 
-    pub fn fill_rects(&mut self, rects: &[Option<Rect>], color: pixels::Color) -> Result<(), String> {
+    pub fn fill_rects(&mut self, rects: &[Rect], color: pixels::Color) -> Result<(), String>
```

Comment:
A decision was made for this signature.  the into function on the Into trait consumes the value it is being called on, which makes transforming this signature difficult.  Given the low likelihood of a slice of Option<Rect> actually being necessary I chose to change this to be a slice of Rect.